.github/CONTRIBUTING.md

```diff
@@ -31,27 +34,33 @@ If you're developing against Magic Modules, vendoring changes is done automatica
 ### Running Tests
 
 The following environment variables must be set in order to run tests:
+
 ```
 GOOGLE_PROJECT
-GOOGLE_CREDENTIALS or GOOGLE_USE_DEFAULT_CREDENTIALS
+GOOGLE_CREDENTIALS
```

Comment:
This might be obvious to TF users, but can you specify that this should be set to a JSON service account creds file.

Reply:
The primary place for configuration information is the provider reference and I'm not really interested in homing that information here; https://www.terraform.io/docs/providers/google/provider_reference.html#configuration-reference

This is actually a subset of the fields that work and I'm fairly certain any set of provider env vars actually works, so once someone's read that it will be clear what the values they need are.